lighthouse-core/audits/uses-rel-preload.js

```diff
@@ -112,7 +112,6 @@ class UsesRelPreloadAudit extends Audit {
       const originalNode = originalNodesByRecord.get(node.record);
       const timingAfter = simulationAfterChanges.nodeTimings.get(node);
       const timingBefore = simulationBeforeChanges.nodeTimings.get(originalNode);
-      // @ts-ignore TODO(phulce): fix timing typedef
       const wastedMs = Math.round(timingBefore.endTime - timingAfter.endTime);
```

Comment:
ha, looks like the `@ts-ignore` was also hiding the possibly undefined (to tsc) response from `nodeTimings.get()`. I'm good with casting or a runtime check. It would be nice to avoid `@ts-ignore` since it masks everything in the line, but it also wouldn't be the end of the world

Reply:
whoops yeah, added a check for it! 👍